requirements.txt

```diff
@@ -1,4 +1,17 @@
-Django~=2.0.3
-django-reversion==2.0.13
+certifi==2019.3.9
+chardet==3.0.4
+Django==2.0.13
 django-bootstrap3==9.1.0
+django-reversion==2.0.13
+entrypoints==0.3
+flake8==3.7.7
 gunicorn==19.7.1
+idna==2.8
+mccabe==0.6.1
+pbr==5.1.3
+pkg-resources==0.0.0
+pycodestyle==2.5.0
+pyflakes==2.1.1
```

Comment:
I'm not sure we want to add pyflakes (and its dependencies) here. Otherwise that means that it will be installed on the production as well.

Reply:
Yes I was not sure either; reverted it